traits/core.md

```diff
@@ -0,0 +1,91 @@
+# Core traits
+
+## Manual Scaler
+
+This trait definition allows operators to manually scale the number of replicas for workload types that allow scaling operations.
+
+### Applicable Workloads
+
+ - `core.oam.dev/v1alpha2.ContainerizedWorkload`
+
+### Schematic
+
+| Attribute | Type | Required | Default Value | Description |
+|-----------|------|----------|---------------|-------------|
+| `replicaCount` | `integer` | Y | `0` | The target number of replicas to create for a given component instance. |
+
+### Spec
+
+As a core trait, we don't need any trait definition here. But we could give this as an example.
```

Comment:
what does this mean?

Reply:
@ryanzhang-oss 
Talked with Jianbo offline. I will help fix the grammars and expression. Will push to his branch later.